src/ralph/accounts/views.py

```diff
@@ -28,9 +38,70 @@ def user_licence(self, item):
             return []
 
 
+class InventoryTagConfirmationView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_tag_confirmation.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        asset_fields = [
+            ('barcode', _('Barcode / Inventory Number')),
+            'model__category__name', 'model__manufacturer__name',
+            'model__name', ('sn', _('Serial Number'))
+        ]
+
+        context['asset_details'] = MyEquipmentAssetList(
+            BackOfficeAsset.objects.filter(id=self.kwargs['asset_id']),
+            asset_fields,
+            request=self.request
+        )
+        return context
+
+
+class InventoryAssetMissingView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_missing.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['settings'] = settings
+        return context
+
+
+class InventoryTagView(View):
+    http_method_names = ['post']
+
+    def post(self, request, *args, **kwargs):
+        asset = get_object_or_404(BackOfficeAsset, id=request.POST['asset_id'])
+        if not asset.user_id == request.user.id:
+            return HttpResponseForbidden()
+
+        date_tag = None
```

Comment:
I meant that this is enough
`date_tag = settings.INVENTORY_TAG + '_' + date.today().isoformat()`
AFAIK